examples/tfengine/full.hcl

```diff
@@ -33,7 +33,7 @@ data = {
 }
 
 template "devops" {
-  recipe_path = "{{$base}}/org/devops.hcl"
+  recipe_path = "{{$base}}/devops.hcl"
```

Comment:
Nit: Looking at it again, I think it would be clearer if the `$base` variable is named `$recipes`.

Reply:
I like that. Changed